src/main/java/org/usfirst/frc4904/robot/RobotMap.java

```diff
@@ -86,8 +173,51 @@
+        Component.canCoderConfiguration = new CANCoderConfiguration();
+        Component.leftWheelEncoder.configAllSettings(Component.canCoderConfiguration);
+        Component.rightWheelEncoder.configAllSettings(Component.canCoderConfiguration);
+        Component.leftWheelEncoder.configFeedbackCoefficient(RobotMap.Metrics.Chassis.METERS_PER_TICK, "meters",
+                SensorTimeBase.PerSecond);
+        Component.rightWheelEncoder.configFeedbackCoefficient(RobotMap.Metrics.Chassis.METERS_PER_TICK, "meters",
+                SensorTimeBase.PerSecond);
+
+        Component.leftWheelAccelerationCap = new EnableableModifier(new AccelerationCap(Component.pdp));
+        Component.leftWheelAccelerationCap.enable();
+        Component.rightWheelAccelerationCap = new EnableableModifier(new AccelerationCap(Component.pdp));
+        Component.rightWheelAccelerationCap.enable();
+
+        Component.leftDriveA = new Motor("leftDriveA", false, Component.leftWheelAccelerationCap,
+                new CANTalonFX(Port.CANMotor.LEFT_DRIVE_A));
+        Component.leftDriveB = new Motor("rightDriveA", false, Component.rightWheelAccelerationCap,
+                new CANTalonFX(Port.CANMotor.RIGHT_DRIVE_A));
+        Component.rightDriveA = new Motor("leftDriveB", false, Component.leftWheelAccelerationCap,
+                new CANTalonFX(Port.CANMotor.LEFT_DRIVE_B));
+        Component.rightDriveB = new Motor("rightDriveB", false, Component.rightWheelAccelerationCap,
+                new CANTalonFX(Port.CANMotor.RIGHT_DRIVE_B));
+
+        Component.turnPID = new CustomPIDController(PID.Turn.P, PID.Turn.I, PID.Turn.D, PID.Turn.F,
+                (PIDSensor) Component.navx);
+        Component.shifter = new SolenoidShifters(Port.Pneumatics.SHIFTER.buildDoubleSolenoid());
+
+        Component.chassis = new TankDriveShifting(0.0, RobotMap.Component.leftDriveA, RobotMap.Component.leftDriveB,
+                RobotMap.Component.rightDriveA, RobotMap.Component.rightDriveB, Component.shifter);
+
+        Component.splinesChassis = new SensorDrive(Component.chassis, AutoConstants.AUTO_CONSTANTS,
+                DriveConstants.DRIVE_CONSTANTS, Component.leftWheelEncoder, Component.rightWheelEncoder,
+                Component.navx);
+
         HumanInput.Driver.xbox = new CustomXbox(Port.HumanInput.xboxController);
         HumanInput.Operator.joystick = new CustomJoystick(Port.HumanInput.joystick);
+        NetworkTables.inst = NetworkTableInstance.getDefault();
+        NetworkTables.Vision.table = NetworkTables.inst.getTable("vision");
+        NetworkTables.Vision.xDistanceToTarget = NetworkTables.Vision.table.getEntry("xDistanceToTarget");
+        NetworkTables.Vision.yDistanceToTarget = NetworkTables.Vision.table.getEntry("yDistanceToTarget");
+        NetworkTables.Vision.theta = NetworkTables.Vision.table.getEntry("theta");
+        NetworkTables.Vision.beta = NetworkTables.Vision.table.getEntry("beta");
```

Comment:
This is a bit static and not how our pipeline works. Can networktables store a list of items? The pipeline generates multiple [`Target`](https://github.com/RoboticsTeam4904/standard-vision/blob/a2cd5552b63bb1bb9ed23f8a8eaeb7f476c15b0a/stdvis-core/src/types.rs#L78-L85)s, which have the 4 measurement values plus an identifier for which type of target it is. Ideally the robot code picks which target it's going to use; this would be as simple as picking the nearest target where `id = 0` (the high port target).